www/shared/styles/imports/custom-medias.css

```diff
@@ -4,6 +4,9 @@
    Use it like so: @media (--layout-small) { ... }
    ========================================================================== */
 
+/* this need to be according to the sum of --layout-container-width and 2 * --layout-edge-spacing-container-x */
+@custom-media --layout-lte-container-width (width <= 96.25em); /* 1540px */
```

Comment:
Consider changing this to 
```
@custom-media --layout-lte-container-width (width <= calc(var(--layout-container-width) + calc(2 x  var(--layout-edge-spacing-container-x)) );
```

I haven't tested it tho, just a suggestion 😅 
```

Reply:
`--layout-edge-spacing-container-x` is a percentage. 
And if I'm not mistaken, variables don't work on this file.

Reply:
@nlfonseca didn't we change `--layout-edge-spacing-container-x` from 6% to 6vh?

Reply:
That's what I thought, but can you test with the build script and see if the `calc` plugin works? 😄

Reply:
custom-media don't accept variables, so calc doesn't work either.
I will put a warning on these variables to update this custom-media in case of change.